gms/feature_service.hh

```diff
@@ -61,8 +67,28 @@ public:
     void enable(const sstring& name);
     void enable(const std::set<std::string_view>& list);
     db::schema_features cluster_schema_features() const;
+
+    // The set of features this node supports.
+    // It is only legal to enable a feature once every node in the cluster supports it.
+    //
+    // The set of supported features may grow while the node is running (e.g. when node configuration is reloaded).
```

Comment:
Can it only grow? Can't we "lose" a feature after a reload? Or we decree that this is an error - just like if I remember correctly a node cannot join a cluster if it already supports a feature that the new node doesn't know to support.
If this is the case, then it would be nice to add a line to the comment explicitly saying that the set of features cannot shrink, because ...